snapshottest/module.py

```diff
@@ -17,6 +18,14 @@ def _escape_quotes(text):
     return text.replace("'", "\\'")
 
 
+def _load_source(module_name, filepath):
```

Comment:
```suggestion
def _load_source(module_name, filepath):
    """
    Replaces old imp.load_source() call.
    
    The imp module was dropped in Python 3.12 in favor of the importlib.
    See: https://docs.python.org/3.11/library/imp.html#imp.load_module
    
    Following code was inspired by the importlib documentation example: 
    https://docs.python.org/3.12/library/importlib.html#importing-a-source-file-directly
    """
```

Reply:
Could you link the mailing list thread too?

Reply:
@paulmelnikow updated the description :+1: